.jvmopts

```diff
@@ -0,0 +1,4 @@
+-Xms3784m
+-Xmx3784m
+-XX:MaxMetaspaceSize=1g
```

Comment:
failed Java 7 build

https://travis-ci.org/scalaz/scalaz/jobs/313932269#L635

```
Unrecognized VM option 'MaxMetaspaceSize=1g'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

Reply:
what about this comment?

```
  - oraclejdk8 # scalaz 7.3 does not support Java 7
```